gallery.php

```diff
@@ -54,7 +54,7 @@
 				<label data-l10n="sendAllMail" for="mail-form-send-link"></label>
 			<?php endif; ?>
 
-			<button class="mail-form-input btn" name="submit" type="submit" value="Senden">Senden</button>
+			<button class="mail-form-input btn" name="submit" type="submit" value="Send">Send</button>
```

Comment:
That should be translatable.

Reply:
Right, I'll take a look in the evening or tommorow